x-pack/plugin/src/test/resources/rest-api-spec/test/runtime_fields/90_loops.yml

```diff
@@ -183,6 +183,7 @@ setup:
             loop:
              terms:
                field: over_max_depth
```

Comment:
do you mind adding one test here also for loops around queries? I know you wrote a lot of those above (thanks) but I would find this yaml suite more complete if we also covered loops for queries (I did not add specific tests in the first place because I knew they would fail)

Reply:
I've added some in the last commit I pushed.

Reply:
thanks!